.github/workflows/release.yml

```diff
@@ -44,10 +44,13 @@ jobs:
           VERSION=$(grep -E '^FROM ubuntu:[a-z]+-[0-9]+$' Dockerfile)
           VERSION=${VERSION##*:}
           STREAM=${VERSION%-*}
+          if [[ -z "$STREAM" ]]; then
+            STREAM=$VERSION
+          fi
```

Comment:
Not needed for this but trying to unify a bunch of docker release workflows so they can be made into an action.